### PR TITLE
refactor: remove open-browser code duplication

### PR DESCRIPTION
**Pull request by erezrokah**
**- Summary**

Fixes https://github.com/netlify/cli/issues/1212, follow up on https://github.com/netlify/cli/pull/1144

More context - not only we had duplicate code around this, we were using 2 different dependencies to do so. I removed the usage of https://github.com/oclif/cli-ux/blob/515570bbeb40bcd28698b6b7af53cc9bfdf86279/src/open.ts#L12 as it seems like an out of date version of https://github.com/sindresorhus/open/blob/f3748e4f5a04a41e27034aff1eb08e8b86d01c1c/index.js#L34

**- Test plan**

Tested manually the different commands related to opening browsers (`login,dev,deploy,init,open,sites:create --with-ci`).

**- Description for the changelog**

Remove open browser code duplication

**- A picture of a cute animal (not mandatory but encouraged)**

🐕

### CODE REVIEW

File: src/commands/dev/index.js [hunk 1]
```diff
@@ -223,7 +213,9 @@ class DevCommand extends Command {
     const liveTunnelUrl = await handleLiveTunnel({ flags, site, api, settings, log })
     url = liveTunnelUrl || url
 
-    await openBrowser({ devConfig, url, warn })
+    if (devConfig.autoLaunch !== false) {
+      await openBrowser({ url, log, silentBrowserNoneError: true })
```
Comment (erezrokah) [inline src/commands/dev/index.js:217]: In all other cases we print an error when `BROWSER=none` is set as an env variable. When running `netlify dev` is it quite useful to run `BROWSER=none netlify dev` and should not be considered as an error.

File: src/utils/open-browser.js [hunk 2]
```diff
@@ -1,31 +1,32 @@
...
 const isDockerContainer = require('is-docker')
 
-function unableToOpenBrowserMessage(url, err) {
-  // https://github.com/sindresorhus/log-symbols
-  console.log('---------------------------')
-  const errMsg = err ? `\n${err.message}` : ''
-  const msg = `Error: Unable to open browser automatically${errMsg}\n`
-  console.log(`${chalk.redBright(msg)}`)
-  console.log(chalk.greenBright('Please open your browser & open the URL below to login:'))
-  console.log(chalk.whiteBright(url))
```
Comment (erezrokah) [inline src/utils/open-browser.js:12]: The choice of colours here made the message almost unreadable (`whiteBright`).

File: src/utils/open-browser.js [hunk 3]
```diff
@@ -1,31 +1,32 @@
-const cli = require('cli-ux').default
```
Comment (erezrokah) [inline src/utils/open-browser.js:1]: We use `cli-ux` in some other locations so we can remove this

**Review by ehmicky — APPROVED** [review]
:tada:

**Pull request merged**